### DISCUSSION TITLE
Github Edu Benifits

### DISCUSSION THREAD

**Discussion by abhishekx3** [community/community#147656] [GitHub Education]
### Hi everyone,

Approved on 18th Dec, still not able to use the benefits, kindly help

**Comment by davevad93** [comment]
Hi @abhishekx3 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @abhishekx3 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by samus-aran** [reply]
Love this answer @davevad93 ❤️ Let me pick your brain 🤔 How do you think we could improve our FAQs to help folks with asking them to wait for at least 72 hours?

**Reply by davevad93** [reply]
Hey @samus-aran 😃 , well honestly I think this topic is already well explained in the FAQs, for example; "`We aim to provide you with your educational benefits within 3 days of this approval status`" or "`If you have not received your education benefits within 5 days of your approval status please raise a discussion on the GitHub Education community`". In my opinion, the problem is that most users aren't even aware of the FAQs in the first place, so I think an improvement would be to make the FAQs  more visible somehow or create a sticky post here in the Community just on the topic "`How long after I've been approved will I receive my academic benefits?`" , since most of the discussions regarding GitHub Education are related to this.
